### ISSUE TITLE
Layout/IndentationWidth is confused by conditional class declaration

### ISSUE THREAD

**Issue by rhys-vdw** [rubocop-hq/rubocop#6243]
Rule misbehaves.

--------

## Expected behavior

Permit the following:

```rb
Rails.env.development? && class ManualTesting
  private_class_method def self.routes
    Rails.application.routes.url_helpers
  end
```

## Actual behavior

```
app/services/manual_testing.rb:2:3: C: Layout/IndentationWidth: Use 2 (not -24) spaces for indentation.
  private_class_method def self.routes
  ^^^^^^^^^^^^^^^^^^^^^^^^
```

## Steps to reproduce the problem

Lint the provided code

## RuboCop version

```
-> bundle exec rubocop -V
0.58.2 (using Parser 2.5.1.2, running on ruby 2.4.4 x86_64-darwin17)
```
